.storybook/preview-body.html

```diff
@@ -12,5 +12,7 @@
     margin: 0;
     font-size: 14px;
     width: 100%;
+    overflow-x: auto;
+    overflow-y: auto;
```

Comment:
Without this, we don't get scrollbars when the documentation is too large to fit in the browser window.